Code/6-wire-version/data/wifi.html

```diff
@@ -34,15 +34,15 @@ <h2>Access Point:</h2>
 			<table>
 				<tr>
 					<td><label for="enableAp">Enable specific AP:</label></td>
-					<td><input type="checkbox" id="enableAp" disabled></td>
+					<td><input type="checkbox" id="enableAp"></td>
 				</tr>
 				<tr>
 					<td>SSID:</td>
 					<td><input type="text" id="apSsid"></td>
 				</tr>
 				<tr>
 					<td>Password:</td>
-					<td><input type="text" id="apPwd"></td>
+					<td><input type="password" id="apPwd"><input type="checkbox" onclick="togglePlainText('apPwd')"></td>
```

Comment:
this does not show any password in clear text (if you don't want - toggle with the checkbox).
i think there should not be any replacements in the input field since we would have to enter the password each time we edit anything else

Reply:
Hidden passwords wont protect against targeted attacks, but I think a realistic goal is to not show passwords more than necessary. Like friends and neighbours that may have access to the wifi don't need to see the mqtt login. I think it's okay to enter new credentials every time I change network or mqtt. How often do you need to change it really? So the only time we should need to send it unecrypted over the network is when we send it TO the ESP. My opinion is that I wanna see what I type and then it disappears after posting. But I'm open for other opinions.

Reply:
ok, agreed. added to the config:
```cpp
/** get the state of password visibility */
const bool hidePasswords = true;
```